AGENTS.md

```
@@ -0,0 +1,172 @@
+make                # Build, test, and lint (default target)
+make go-test        # Unit tests only (excludes e2e via build tags)
+make test-e2e       # E2E tests (requires live cluster + KUBECONFIG)
+make generate       # Regenerate deepcopy, CRDs, OpenAPI
+make manifests      # Regenerate CRD YAML and RBAC
+make lint           # Run linters
+make coverage       # Coverage report
+```
+
+## Repository Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)